### ISSUE TITLE
I can't close lightbox if all images fail to load.

### ISSUE THREAD

**Issue by dviluk** [andreknieriem/simplelightbox#250]
Hi, I'm having some problems.

If all images fail to load (404) the lightbox can't get closed. I don't know what I'm doing wrong.

This is my code:

```js
var lightbox = new window.SimpleLightbox('.preview-image')
```

```html
<a href="http://example.example/image_not_found.png" target="_blank" class="preview-image">
    <i class="fa fa-download"></i>
</a>
```

When debugging in this part it says `this.isOpen = false` when all images fail to load.

```js
// class SimpleLightbox.close()
close() {
    if (!this.isOpen || this.isAnimating || this.isClosing) {  // this.isOpen = false
        return false; // and breaks here
}
```

Thanks!

**Comment by dviluk** [comment]
Here is set `this.isOpen = false`

```js
// class SimpleLightbox.adjustImage()
...
tmpImage.addEventListener('error', ...)
```

**Comment by andreknieriem** [comment]
Thanks for mention. I will fix this

**Comment by andreknieriem** [comment]
Fixed in 2.8.1
